wp-content/plugins/cp-gp-sanitize/cp-gp-sanitize.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Plugin name: ClassicPress GlotPress Sanitize strings
+ * Description: Sanitize the strings content to strip specific HTML tags
+ * Version:     1.0
+ * Author:      ClassicPress.net
+ * Author URI:  http://classicpress.net
+ * License:     GPLv2 or later
+ */
+
+class CP_GP_Sanitize {
+
+	function __construct() {
+		add_action( 'gp_translation_prepare_for_save', array( $this, 'apply_sanitize' ), 2, 99999 );
+	}
+
+	function apply_sanitize( $args, $translation ) {
+		foreach( range( 0, $translation->get_static( 'number_of_plural_translations' ) - 1 ) as $i ) {
+			if ( isset( $args[ "translation_$i" ] ) ) {
+				$temp = $this->sanitize( $args[ "translation_$i" ] );
+				
+				if( $args[ "translation_$i" ] !== $temp ) {
+					error_log( 'User ' . get_current_user_id() . ': ' . print_r( $args, true ) );
```

Comment:
I haven't forgotten about this PR, just been busy with other things.

- Right now, the `error_log` message doesn't really provide any information about what this is or where it came from.
- `print_r` will print across multiple lines which doesn't work on Apache at least.  We also need this to be a single line so we can analyze it later.

This should work better:

```
error_log(
    'Invalid translation by user ID ' . get_current_user_id() . ': '
    . json_encode( $args, JSON_UNESCAPED_SLASHES )
);
```

Also: what error lines does this actually produce?  Is there a way we can test it and see the results before we merge it?